pageserver/src/page_service.rs

```diff
@@ -1080,6 +1087,10 @@ impl PageServerHandler {
             }
         };
 
+        let _timer = timeline
+            .query_metrics
+            .start_timer(metrics::SmgrQueryType::GetPageAtLsn);
+
```

Comment:
The measure doesn't include the time it took to get the cached page.
I don't think it's a big deal, but, would have been nice to avoid it.

Reply:
You mean get the cached timeline?  I think we're still covering the part where we would hit a cached page.

Reply:
Yeah, sorry, I meant "cached struct Timeline" :(